client/components/emojify/index.jsx

```diff
@@ -47,8 +47,14 @@ export default class Emojify extends PureComponent {
 	}
 
 	render() {
+		const {
+			children,
+			className, // eslint-disable-line no-unused-vars
```

Comment:
It's poorly named, but the `className` prop is applied to the Twemoji `<img>`, applying it to the container `<div>` sounds like a recipe for weird clashing CSS rules.